server/config/config.go

```diff
@@ -432,7 +432,7 @@ func (c *Configurator) GetAPIConfig() *APIConfig {
 }
 
 func (c *Configurator) GetGithubConfig() *GithubConfig {
-	if c.gc.Github.ClientID == "" {
+	if c.gc.Github == (GithubConfig{}) {
```

Comment:
Yikes, sorry about that - I think the PAT support got a little lost in the recent config refactor.